exchange/client.go

```diff
@@ -13,9 +14,18 @@ func BuildHTTPClient(options *Options) (*http.Client, error) {
 		checkRedirect = nil
 	}
 
+	transp := http.DefaultTransport.(*http.Transport).Clone()
+	transp.TLSClientConfig.InsecureSkipVerify = options.SkipVerify
+	if options.ForceHTTP1 {
+		transp.TLSClientConfig.NextProtos = []string{"http1"}
```

Comment:
`http1` does not seem to be correct.
I looked at Go source code, but  could not find `http1` used as a value of `NextProtos`.
Perhaps `http/1.1` is right.

I investigated the semantics of `NextProtos` and `TLSNextProto`, and how these configurations affects others.
But the configurations around HTTP and TLS were too complicated to understand.
I'm nervous whether this change is breaking anything I don't expect.

So I want to simply drop `--http1` flag, and allow Go runtime to choose protocol.

Reply:
As previously mentioned, I used this switch to test envoy proxy and this was really useful. The go documentation is not really clear but: "Programs that must disable HTTP/2 can do so by setting Transport.TLSNextProto (for clients)" was not enough so I had to force the Nextproto something different then "h2".